services/go_app_modules/authentication-api/ressources/auth/service/userinfo_model.go

```diff
@@ -0,0 +1,12 @@
+package service
+
+type AuthentikUserInfo struct {
+	ID                string   `json:"sub"`
+	Email             string   `json:"email"`
+	EmailVerified     bool     `json:"email_verified"`
+	Name              string   `json:"name"`
+	DisplayName       string   `json:"given_name"`
+	PreferredUsername string   `json:"preferred_username"`
+	Nickname          string   `json:"nickname"`
+	Groups            []string `json:"groups"`
+}
```

Comment:
can you do an example? These are just here to transfer data from storage to http layer, or between service layer functions

Reply:
It would make sense to use such a validator for the http layer model, when a user submits data. But at the moment it is read only code and this is a transportation model within services, so no user input involved